fix(@angular/cli): added clarity on angular cli version issues

johnpapa opened the pull request:
fixes https://github.com/angular/angular-cli/issues/16885

alan-agius4 commented:
Thanks for this, can you please amend the commit msg header to something like `fix(@angular/cli): added clarity on outdated message`?

Thanks

johnpapa commented:
> Thanks for this, can you please amend the commit msg header to something like `fix(@angular/cli): added clarity on outdated message`?
> 
> Thanks

Modified. thanks
